vmm/sandbox/src/vm.rs

```diff
@@ -197,3 +198,7 @@ impl FromStr for ShareFsType {
         }
     }
 }
+
+pub struct VcpuThreads {
+    pub vcpus: HashMap<isize, isize>,
```

Comment:
why map? what is the key used for?

Reply:
Because in some scenarios, vCPU thread index is needed for binding the vcpu thread of VM to host physical core.